packages/lit/build.sh

```diff
@@ -0,0 +1,25 @@
+TERMUX_PKG_HOMEPAGE=https://luvit.io
+TERMUX_PKG_DESCRIPTION="Toolkit for developing, sharing, and running luvit/lua programs and libraries."
+TERMUX_PKG_LICENSE="Apache-2.0"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=3.8.5
+TERMUX_PKG_SRCURL=git+https://github.com/luvit/lit.git
```

Comment:
We aren't using a release tarball because they don't contain the git submodules?

Reply:
Yes.